src/Shared/ExceptionHandling.cs

```diff
@@ -281,8 +283,8 @@ internal static bool NotExpectedSerializationException(Exception e)
         /// </summary>
         internal static bool NotExpectedRegistryException(Exception e)
         {
-            if (e is SecurityException
-             || e is UnauthorizedAccessException
+            if (e is UnauthorizedAccessException
+             || e is SecurityException
```

Comment:
why this swap?

Reply:
Because I moved stuff around and forgot to put it back the way it was.